paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetActivityTest.kt

```diff
@@ -40,37 +42,76 @@ internal class CustomerSheetActivityTest {
     fun `Finish with cancel on back press`() {
         runActivityScenario { scenario ->
             scenario.onActivity {
+                composeTestRule.waitForIdle()
```

Comment:
Why are these needed?

Reply:
Need to wait for animations to complete.